packages/react-components/react-migration-v8-v9/src/components/Theme/v9ThemeShim.ts

```diff
@@ -206,8 +206,7 @@ const mapBorderRadiusTokens = (effects: IEffects): Partial<BorderRadiusTokens> =
  * You can optional pass a base v9 theme; otherwise webLightTheme is used.
```

Comment:
```suggestion
 * You can optional pass a base v9 theme; otherwise webLightTheme or webDarkTheme is used, depending on whether the v8 theme is inverted
```